lib/kubeclient/common.rb

```diff
@@ -51,6 +53,9 @@ def initialize_client(
       validate_auth_options(auth_options)
       handle_uri(uri, path)
 
+      @class_owner = class_owner
+      @entities = nil
```

Comment:
@moolitayer I wasn't suggesting to remove it, I was suggesting to use the initialization value that you need (empty hash).